src/multibody/fcl.hxx

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright (c) 2015-2016 CNRS
+//
+// This file is part of Pinocchio
+// Pinocchio is free software: you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation, either version
+// 3 of the License, or (at your option) any later version.
+//
+// Pinocchio is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Lesser Public License for more details. You should have
+// received a copy of the GNU Lesser General Public License along with
+// Pinocchio If not, see
+// <http://www.gnu.org/licenses/>.
+
+#ifndef __se3_fcl_hxx__
+#define __se3_fcl_hxx__
+
+
+#include <iostream>
+
+
+namespace se3
+{
+
+  inline std::ostream & operator << (std::ostream & os, const CollisionPair & X)
+  {
+    X.disp(os); return os;
+  } 
+  
+
+#ifdef WITH_HPP_FCL  
+
+
+  inline double DistanceResult::distance () const
+  {
+    return fcl_distance_result.min_distance;
+  }
+
+  inline Eigen::Vector3d DistanceResult::closestPointInner () const
+  {
+    return toVector3d(fcl_distance_result.nearest_points [0]);
+  }
+
+  inline Eigen::Vector3d DistanceResult::closestPointOuter () const
+  {
+    return toVector3d(fcl_distance_result.nearest_points [1]);
+  }
+    
+  inline bool operator == (const fcl::CollisionObject & lhs, const fcl::CollisionObject & rhs)
+  {
+    return lhs.collisionGeometry() == rhs.collisionGeometry()
+            && lhs.getAABB().min_ == rhs.getAABB().min_
+            && lhs.getAABB().max_ == rhs.getAABB().max_;
```

Comment:
The operator== is necessary for structures/class that we want to bind in boost:python as vectors.
See [this SO](http://stackoverflow.com/a/10682380)